scripts/generate-clientset.sh

```diff
@@ -32,7 +32,7 @@
 # Alternatively, make a symlink from the GOPATH package source directory to the
 # repo directory.
 
-set -euox pipefail
+set -euo pipefail
```

Comment:
Why removing -x?

Reply:
it makes the output a lot more verbose than it needs to be IMO, and the extra verbosity doesn't seem to add much value

Reply:
Fair, thanks!